### ISSUE TITLE
JSON format functional broke at ver X37.4

### ISSUE THREAD

**Issue by vladimirfilin** [visualfc/liteide#1194]
### What version of LiteIDE are you using (LiteIDE About - Version and Build Qt Version)?
X37.4 Windows 10 x64
Qt 5.14.2 (64 bit)

### What version of Go are you using (`go version`)?
go version go1.16 windows/amd64

### What operating system and processor architecture are you using(`go env`)?
set GO111MODULE=off
set GOARCH=amd64
set GOBIN=
set GOCACHE=C:\Users\Vovka\AppData\Local\go-build
set GOENV=C:\Users\Vovka\AppData\Roaming\go\env
set GOEXE=.exe
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOINSECURE=
set GOMODCACHE=C:\Gopath\pkg\mod
set GONOPROXY=
set GONOSUMDB=
set GOOS=windows
set GOPATH=C:\Gopath
set GOPRIVATE=
set GOPROXY=https://proxy.golang.org,direct
set GOROOT=c:\go
set GOSUMDB=sum.golang.org
set GOTMPDIR=
set GOTOOLDIR=c:\go\pkg\tool\windows_amd64
set GOVCS=
set GOVERSION=go1.16
set GCCGO=gccgo
set AR=ar
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -fmessage-length=0 -fdebug-prefix-map=C:\Temp\go-build3728854670=/tmp/go-build -gno-record-gcc-switches

### What did you do?
Looks like json format functional  #141 is broken now. I tried to format json:
```json
{
       "scriptId": 100,
       "commands": [
       {
                "commandId": 100,
                "command": "command 1",
                "position": 1
        }]}
```
`JsonEdit plugin v.X27 jsonedit.dll` is enabled.` json.xml` file is exists.

### What did you expect to see?
Formatted json, something like this:
```json
{
       "scriptId": 100,
       "commands": [
        {
                "commandId": 100,
                "command": "command 1",
                "position": 1
        }
      ]
}
```

### What did you see instead?
- `Verify` command do nothing
- `Format Json` command returns `jsonfmt: ` to Event Log
- `Compact Json` command returns `jsonfmt: ` to Event Log

**Comment by visualfc** [comment]
fixed  https://github.com/visualfc/gotools/commit/e23d62dc73fe0daaddec189debf8b863addfdfaf
please get & update gotools <https://github.com/visualfc/gotools>

Windows/Linux: copy GOPATH/bin gotools and gocode to liteide/bin
MacOS: copy GOPATH/bin gotools and gocode to LiteIDE.app/Contents/MacOS

**Comment by vladimirfilin** [comment]
Everything works perfectly! Bug fixes at the \gotools [PR #14](https://github.com/visualfc/gotools/pull/14)

**Comment by vladimirfilin** [comment]
Done
